packages/plugin-core/src/commands/CreateNoteWithTraitCommand.ts

```diff
@@ -141,7 +141,7 @@ export class CreateNoteWithTraitCommand extends BaseCommand<
       }
     }
 
-    await new GotoNoteCommand().execute({
+    await new GotoNoteCommand(getExtension()).execute({
```

Comment:
Since we already have getExtension() imported into these classes it doesn't make it worse to use it for now, but it also doesn't make it better as time goes on these should be refactored to use extension that gets passed into the instance of the command.

(or if something is used statically that we dont want to refactor right away then it should use ExtensionProvider to grab the interface of DendronExtension rather than coupling ourselves to concrete implementation of DendronExtension).